sdk/communication/azure-communication-rooms/swagger/README.md

```diff
@@ -21,7 +21,7 @@ To update generated files for rooms service, run the following command
 ### Code generation settings
 ``` yaml
 require:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/af0e925f435eed808cfa0168057405a43991c7ab/specification/communication/data-plane/Rooms/readme.md
+    - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/communication/data-plane/Rooms/readme.md
```

Comment:
Did you regenerate the code using the swagger? I thought we need the raw readme file in order to generate